cypress/integration/spec_multiuser_sessions.js

```diff
@@ -0,0 +1,92 @@
+describe('Test the actions in UTXO list', () => {
```

Comment:
This is probably from copy&pasting, change this to provide more info on what the test is about.